gui-apps/lavalauncher/lavalauncher-1.7.1.ebuild

```diff
@@ -0,0 +1,31 @@
+# Copyright 2019-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
```

Comment:
Please take a look at the upstream meson config (add_project_arguments) and ensure that no additional GCC options are passed. We must respect users settings... unless justified. If we need to add those then we also need to ensure things are appended properly.

Reply:
Sorry, I am a bit unsure what you are trying to say?

Do you mean like -O3 or those things?

Reply:
Sorry am not too familiar with gentoo C++ ideologies and intricacies.

Reply:
Ah, I think I figured it out, with the `-werror=true` and removed that. :smile_cat:

Reply:
done!